pandas/core/algorithms.py

```diff
@@ -1502,7 +1504,12 @@ def take(
 # ------------ #
 
 
-def searchsorted(arr, value, side="left", sorter=None) -> np.ndarray:
+def searchsorted(
+    arr: ArrayLike,
+    value: NumpyValueArrayLike,
+    side: Literal["left", "right"] = "left",
```

Comment:
this is npt._SortSide? we probably don't want to use private aliases in the codebase, but maybe OK to do so in pandas._typing. thoughts?

Reply:
Can't use `_SortSide` from numpy as it is only defined in their `pyi` files